hack/mock-unittest-data/gomod.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+import json
+import subprocess
+from contextlib import contextmanager
+from functools import partial
+from itertools import chain
+from os import environ
+from pathlib import Path
+from shutil import copyfile
+from shutil import rmtree as delete_directory_with_contents
+from sys import argv
+from tempfile import gettempdir
+
+from yarn import print_banner
+
+
+def mkdir_p(dirname):
```

Comment:
nitpick: `mkdir_p` is easily optically lost IMO, would `mkdir_w_parents` work better?